bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/i18n/netatmo.properties

```diff
@@ -0,0 +1,332 @@
+# binding
```

Comment:
When I run the maven tool to produce this file, I find some differences.
Could you please have a look.

Reply:
Fixed in my PR